### PR TITLE
fixed unicode issue (21499) using requests library

### LINKED ISSUE #21499: Python Pandas read_html fails when reading tables from Wikipedia
I am trying to read the tables from a Wikipedia page using the following code:

```python
import pandas as pd
pd.read_html('https://en.wikipedia.org/wiki/2013–14_Premier_League')
```

Doing that generates the following error:

```python
UnicodeEncodeError: 'ascii' codec can't encode character '\u2013' in    position 14: ordinal not in range(128)
```

I have tried

```python
pd.read_html('https://en.wikipedia.org/wiki/2013–14_Premier_League', encoding='utf-8')
```

But still get the same error. The following works:

```python 
import requests
r = requests.get('https://en.wikipedia.org/wiki/2017–18_Premier_League')
c = r.content
dfs = pd.read_html(c)
```
What I want to know is how to get pd.read_html() to work directly on the url without requests. What is it that I don't understand about encoding or is this a problem with Pandas?

I am running an Anaconda distribution of Pandas 0.21.1 and Python 3.5.4. Thanks for any help.

### PR DESCRIPTION
**Pull request by hannah2048**
- [x ] closes #21499 
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Review by hannah2048 — COMMENTED** [review]
added missing newline

**Comment by WillAyd** [comment]
Hmm well it looks like this is going to conflict with the work being done in #21504. @ocefpaf can you see if your PR accounts for this issue (adding a test case, accordingly)? If so would be easier just to bundle this in with your change

**Comment by ocefpaf** [comment]
My guess is that https://github.com/pandas-dev/pandas/pull/21504 could supersede this one. I'll add tests soon. I have a limited time I can work on this. (And I'm waiting for the docstrings questions I asked there before continuing :smile:).

File: pandas/io/html.py [hunk 1]
```diff
@@ -127,8 +127,8 @@ def _read(obj):
     raw_text : str
     """
     if _is_url(obj):
-        with urlopen(obj) as url:
```
Comment (ocefpaf) [inline pandas/io/html.py]: @WillAyd in my PR all `urlopen` calls are wrap with `requests` if it is available so I guess we cover this one in #21504

**Comment by WillAyd** [comment]
Closing in favor of #21504

**Pull request closed without merging**